### ISSUE TITLE
Suggestions Box is not showhingin version 8 for cases.

### ISSUE THREAD

**Issue by BSZ44** [salesagility/SuiteCRM-Core#170]
#### Issue

I've just installed version 8
I've created some knowledgebases
When I open a case, there is no box for suggestions and when I type a subject nothing happens. Of course it used to work like a breeze in 7.xx
I get the same result when trying to reproduce the operation on the suiteCRM 8 demo site

#### Expected Behavior
 A list of suggestions based on relevant knowledgebases  should appear on the cases form

#### Actual Behavior
 nothing 
 

#### Steps to Reproduce

1.
2.
3.
4.

#### Context
 I canoot use version 8 as most of our activity will be cases management

 high as far as I'm concerned

#### Your Environment

* SuiteCRM Version used:
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
* Environment name and version (e.g. MySQL, PHP 7):
* Operating System and version (e.g Ubuntu 16.04):

**Comment by pgorod** [comment]
Wasn't that just your browser's Autofill feature giving you things you had previously typed in that field?

**Comment by pgorod** [comment]
Sorry, I didn't even know about that feature, never used KB... 

But I just tried on the Suite 8 demo and it works.

You should fill the details of "Your environment" properly on this Issue - this makes it easier to try and replicate more accurately.

I suggest trying a different browser to see if it makes any difference. And check the browser's developer console for any errors.

Thanks

**Comment by pgorod** [comment]
It's strange that you can't see it working on the Demo site.... Please give me your _specific_ steps to reproduce: I want to see it with your exact steps, using the same KB article names, typing the same search text , etc

Thanks

**Comment by BSZ44** [comment]
Hi pgorod,

we can see on this screenshot that there is no chance the interface will work considering that there is no "suggestions" field
![Uploading image.png…]()
Basically I'm just creating a couple of knowledgebases with some random text. IT works pretty fine on V7 but not at all on V8.X (I've tried them all) , nor does it work on the demo site.
Fore instance on kb with ATM for title and the second one with BPM. In V7 when I type either of these names the corresponding KB appears.

**Comment by pgorod** [comment]
The suggestion box only appears after you type a few characters and move the cursor outside the subject field... but it works. (on the v8 demo)

**Comment by BSZ44** [comment]
Hi pgorod,

That's exactly what I'm doing.  I've tried that again today both on the suite8 demo and on my local ptf,  both on with Edge and Chrome.  It still doesn't work for me.

**Comment by likhobory** [comment]
Hi @pgorod 

> The suggestion box only appears after you type a few characters and move the cursor outside the subject field... but it works. (on the v8 demo)

I also can't confirm this. It works on v.7.10, but it doesn't work on v.8 - tested on v.8 demo and on my local VM (v.8.2.0.b2 on Ubuntu 22 / php 8.0.20)
On the v.8 the 'suggestion' field is not displayed.

FF 105 / Edge

**Comment by pgorod** [comment]
This is really weird. I'm on Firefox latest... But @likhobory also tried Firefox... I'm confused

Do you guys have any antivirus or JavaScript blockers that could be preventing some code from running?

**Comment by likhobory** [comment]
Yes, I use anivirus and uBlock ext. in FF, but they has been disabled during testing, in any case this feature works on v.7 as expected.
Very strange,. perhaps we are testing different things? Pedro, can you pls add your screenshot here?

**Comment by pgorod** [comment]
Sorry, my bad. I was looking at the v7 demo, thinking I was in the v8. I confirm that on v8 the Suggestion field does not exist.

**Comment by BSZ44** [comment]
Hello pgorod,

the Suggestion field exists when looking at the Cases module in Studio. In labels as well as in layout. There seems to be a glitch with the deployment which does not effect the changes made in studio. The js code for the suggestion logic is not delivered in the various js files as it is in version 7. Maybe this can help you track the issue ?

**Comment by pgorod** [comment]
I guess this was just something that was left out, forgotten, when the new UI was built. You see it in Studio because Studio is back-end and just changes vardefs. But on the UI, it's not the plain record view UI of other modules, it's a special feature of Cases module... and the new screens don't seem have it yet.

**Comment by BSZ44** [comment]
Try to deploy a change made in studio on the edit layout. It doesn't work for me.  I can add and deploy the suggestion box only when I change the detailed view, which is not the way it is supposed to behave.
